src/App.css

```diff
@@ -1,21 +1,285 @@
+@import url('https://fonts.googleapis.com/css2?family=Nunito:wght@300;400;600;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Nunito', sans-serif;
+}
+
+body {
+  background-color: #fafafa;
+}
+
+a {
+  text-decoration: none;
+}
+
 ul {
   list-style-type: none;
 }
 
-.newLinks li {
+.navi-title {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.new-links {
+  align-items: flex-start;
+}
+
+.new-links li {
   margin-right: 10px;
+  align-items: flex-start;
+  text-transform: uppercase;
+  font-family: 'Poppins', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  font-weight: bold;
```

Comment:
- [ ] The font-weight has been declared twice which is not done. Obviously, this is causing the stylelint error. Kindly fix it